### PR TITLE
Matmul benchmarking: case without tile quantization:

### PR DESCRIPTION
**Pull request by shmsong**
This is the benchmarking PR in this series, tracking the resulting performance from this stack of PRs. 

Most recent run on A100:

initial_sweep_0912ext

### CODE REVIEW

**Comment by shmsong** [comment]
most recent check on DGX A100 after the recent round of moving commits around and cleaning up:
3stage0922
4stage0922

File: benchmarks/cpp/nvfuser/CMakeLists.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ if(USE_CUDA)
     softmax_backward.cpp
```
Comment (zasdfgbnm) [inline benchmarks/cpp/nvfuser/CMakeLists.txt]: Note to myself: I have split this file out and merged separately in https://github.com/csarofeen/pytorch/pull/2007
This file is no longer needed here anymore.

File: benchmarks/cpp/nvfuser/matmul.cpp [hunk 2]
```diff
@@ -0,0 +1,356 @@
+#include <torch/csrc/jit/codegen/cuda/arith.h>
```
Comment (zasdfgbnm) [inline benchmarks/cpp/nvfuser/matmul.cpp]: Note to myself: I have split this file out and merged separately in https://github.com/csarofeen/pytorch/pull/2007
This file is no longer needed here anymore.

**Comment by zasdfgbnm** [comment]
After rebasing, this PR is just a trivial PR adding a test, I will merge this now to the bottom of the stack

**Pull request merged**